backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -290,6 +290,67 @@ class TestOperator extends WholeStageTransformerSuite {
     assert(df.queryExecution.executedPlan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
   }
 
+  test("Test round function") {
+    val df = spark.sql("SELECT round(cast(l_orderkey as int), 2) from lineitem limit 1")
+    val result = df.collect()
+    assert(result.length == 1)
+    val expected = Seq(Row(1))
+    TestUtils.compareAnswers(result, expected)
+    df.show()
+    df.explain(false)
+    df.printSchema()
+    assert(df.queryExecution.executedPlan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
+  }
+
+  test("Test greatest function") {
+    val df = spark.sql("SELECT greatest(l_orderkey, l_orderkey) from lineitem limit 1")
+    val result = df.collect()
+    assert(result.length == 1)
+    val expected = Seq(Row(1))
+    TestUtils.compareAnswers(result, expected)
+    df.show()
+    df.explain(false)
+    df.printSchema()
+    assert(df.queryExecution.executedPlan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
+  }
+
+  test("Test least function") {
+    val df = spark.sql("SELECT least(l_orderkey, l_orderkey) from lineitem limit 1")
+    val result = df.collect()
+    assert(result.length == 1)
+    val expected = Seq(Row(1))
+    TestUtils.compareAnswers(result, expected)
+    df.show()
+    df.explain(false)
+    df.printSchema()
+    assert(df.queryExecution.executedPlan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
+  }
+
+  test("Test hash function") {
+    val df = spark.sql("SELECT hash(l_orderkey) from lineitem limit 1")
+    val result = df.collect()
+    assert(result.length == 1)
+    val expected = Seq(Row(-1712319331))
+    TestUtils.compareAnswers(result, expected)
+    df.show()
+    df.explain(false)
+    df.printSchema()
+    assert(df.queryExecution.executedPlan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
+  }
+
+  test("Test get_json_object function") {
```

Comment:
Do we really need to add such test by using tpc-h table. The table is not actually used for get_json_object and we are just doing the test with literal input.

Reply:
Don't have to use tpc-h table.
Have created VeloxFunctionsValidateSuite as new functions UTs.